osmosis/assetlist.json

```diff
@@ -1694,6 +1694,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/RAPTR.png"
         }
       ] 
+    },
+    {
+      "description": "Spitz on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/1570B676D4F35BACD92F470C01E4FD7B19CA7B8D62DA974A76A59079CEE338E3",
```

Comment:
How did you get this ibc denom? It doesn't seem right

Reply:
Hi Jeremy; it's my IBC from my wallet, so i don't understand which IBC you need ?